### DISCUSSION TITLE
Unwanted files in Pycharm

### DISCUSSION THREAD

**Discussion by Teejayssmart** [community/community#141517] [New to GitHub] [answered]
Hello, 
I tried to practice with .ignore file and i ended up with multiple green link on the terminal
![image_2024-10-15_075359579](https://github.com/user-attachments/assets/1ccce60a-c09f-45b5-a020-124a25a0bd63)

**Comment by VirajxD0** [comment, marked as answer]
The green text in the terminal isn't actually an issue 
What you're seeing is the output of creating a new virtual environment using a tool like virtualenv or venv. The green lines indicate new files and directories being created within the virtual environment structure.

**Reply by Teejayssmart** [reply]
Thank you @VirajxD0
